src/components/shared/fab/index.tsx

```diff
@@ -0,0 +1,14 @@
+import Fab, { FabProps } from '@mui/material/Fab';
+
+interface BasicFabProps extends FabProps {
+  children: React.ReactNode;
+  onClick: () => void;
+}
```

Comment:
컴포넌트 파일 내부에서만 사용되는 local 한 Props interface 는
네이밍을 하느라 들어가는 불필요한 시간을 줄이기 위해 Props 라고 네이밍하기로 결정되었던 것 같은데, 확인 부탁드려도 될까요?
```suggestion
interface Props extends FabProps {
  children: React.ReactNode;
  onClick: () => void;
}
```